core/shared/src/main/scala/zio/prelude/AssociativeBoth.scala

```diff
@@ -1200,14 +1200,6 @@ object AssociativeBoth extends LawfulF.Invariant[AssociativeBothDeriveEqualInvar
       def both[A, B](fa: => Vector[A], fb: => Vector[B]): Vector[(A, B)] = fa.flatMap(a => fb.map(b => (a, b)))
     }
 
-  /**
```

Comment:
Did you mean to delete this?

Reply:
Yes, `ZIOAssociativeBoth` can be fully replaced by `ZIOIdentityBoth`. But it needs to be in the `AssociativeBoth` companion object :wink: 
More detailed explanation here: https://meta.plasm.us/posts/2019/09/30/implicit-scope-and-cats/